wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/ConversationSystemMessageCell.swift

```diff
@@ -1046,7 +1046,8 @@ final class ConversationNewDeviceSystemMessageCellDescription: ConversationMessa
     private static func configureForNewClientOfSelfUser(_ selfUser: UserType, clients: [UserClientType], link: URL) -> View.Configuration {
         let string = L10n.Localizable.Content.System.selfUserNewClient(link.absoluteString)
         let attributedText = NSMutableAttributedString.markdown(from: string, style: .systemMessage)
-        let isSelfClient = clients.first?.isEqual(ZMUserSession.shared()?.selfUserClient) ?? false
+        let selfUserClient = SessionManager.shared?.activeUserSession?.selfUserClient
+        let isSelfClient = clients.first?.isEqual(selfUserClient) ?? false
```

Comment:
if possible without too much hassle, inject the UserSession instead

Reply:
In this `CellDescriptors` there is no real view model or view controller around, so it's kind of hard for me..